misc/release.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# release.sh
+#
+# Build a Thentos binary release tarball.
+# Must be run from the base of the thentos project, 
```

Comment:
https://github.com/liqd/thentos/blob/master/thentos-purescript/build.sh#L4 shows how you can make this more robust.